source/module_io/to_wannier90.h

```diff
@@ -79,7 +79,6 @@ class toWannier90
     #ifdef __LCAO
     void init_wannier_lcao(const Grid_Technique& gt,
```

Comment:
how about init_wannier_pw in header file?

Reply:
Oh. I forgot to modify init_wannier_pw but not init_wannier_lcao